README.md

```diff
@@ -764,14 +764,15 @@ in   | cmd.color.get_report       | null       | The command is a request for a
 in   | cmd.color.set              | int_map    | value is a map of color components. val= {"red":200, "green":100, "blue":45}
 out  | evt.color.report           | int_map    | Map of color components, where value is component intensity.
 -|||
-in   | cmd.color.start_transition | object     | Example value: val= {"component": "red", "transition": "up"}
+in   | cmd.color.start_transition | object     | Example value: val= {"component": "red", "transition": "up", "duration": 127}
 in   | cmd.color.stop_transition  | str        | Stop fading/enhancing single color component. val = "red"
 
 #### Service props
 
-Name             | Value example            | Description
------------------|--------------------------|-------------
-`sup_components` | ["red", "green", "blue"] | List of supported color components
+Name             | Value example                                                            | Description
+-----------------|--------------------------------------------------------------------------|-------------
+`sup_components` | ["red", "green", "blue"]                                                 | List of supported color components
+`duration`       | {"minSeconds": 1, "maxSeconds": 127, "minMinutes": 1, "maxMinutes": 127} | Supported duration steps for transition change (optional).
```

Comment:
Wouldn't it be easier to just mention: min: 1, max: 7620? The rounding up to one minute after 127 seems adapter specific and not that important...

Reply:
the idea was to be as specific as possible in here so that the app _can_ provide proper ui that changes the step when needed. 
I think it's better to optionally lose that information in ui than in service because if service doesnt provide it we have no way of knowing about it in ui.
the worst cause I believe would be when an adapter wouldn't support seconds at all, then setting sth small would result in either having 0 or 1 min which would be quite significant

Reply:
This seems a bit odd

The `start_transition` supports passing a duration now, which in the example is `127`.
The `duration` value example seems to indicate that this can be an object with a whole bunch of things.

Question is then, what is the `duration` in the `start_transition`?
Would it not be better to have the duration in min max milliseconds and then have a "steps" property?

Reply:
Also, should this not be `sup_durations`?

Reply:
@bjhandeland You're right. Duration in start_transition interface are seconds. 
Here it should be named "sup_durations" instead of just "durations"! 
Fixing